js_modules/dagit/packages/core/src/graph/SVGViewport.tsx

```diff
@@ -214,8 +214,25 @@ export class SVGViewport extends React.Component<SVGViewportProps, SVGViewportSt
 
   componentDidMount() {
     this.autocenter();
+    // The wheel event cannot be prevented via the `onWheel` handler.
+    document.addEventListener('wheel', this.onWheel, {passive: false});
```

Comment:
This looks good to me -  I think this might be overkill though? Before we merge it it'd be good to test if adding an `event.stopPropagation()` within the old SVGViewportIntercactor.onWheel would fix the problem? It sounds like the event was propagating up from our handler and continuing up to the document + performing the default wheel behavior in addition to ours?

Edit: Ahh OK nevermind I see https://stackoverflow.com/questions/57358640/cancel-wheel-event-with-e-preventdefault-in-react-event-bubbling. Maybe we throw that URL on this line too 😅